.buildkite/test-pipeline.yaml

```diff
@@ -88,11 +88,7 @@ steps:
   - vllm/distributed
   - tests/core
   commands:
-  - VLLM_ALLOW_DEPRECATED_BLOCK_MANAGER_V1=1  pytest -v -s core/test_scheduler.py
-  - VLLM_ALLOW_DEPRECATED_BLOCK_MANAGER_V1=1  pytest -v -s core core/test_chunked_prefill_scheduler.py
-  - VLLM_ALLOW_DEPRECATED_BLOCK_MANAGER_V1=1  pytest -v -s core core/block/e2e/test_correctness.py
-  - VLLM_ALLOW_DEPRECATED_BLOCK_MANAGER_V1=1  pytest -v -s core core/block/e2e/test_correctness_sliding_window.py
-  - pytest -v -s core --ignore=core/block/e2e/test_correctness.py --ignore=core/test_scheduler.py --ignore=core/test_chunked_prefill_scheduler.py --ignore=core/block/e2e/test_correctness.py --ignore=core/block/e2e/test_correctness_sliding_window.py
+  - pytest -v -s core
```

Comment:
I'm not sure if this is split by purpose to support conditional testing. cc @khluu to confirm

Reply:
The PR that turns `pytest -v -s core` to multiple test commands seem to be in PR #9149 from @sroy745 so I think it is probably not for the purpose of supporting conditional testing but I am not sure.

Reply:
I had added that to selectively add the env variable VLLM_ALLOW_DEPRECATED_BLOCK_MANAGER_V1 only to tests which needed it. I added it explicitly to make sure I am not missing any tests which continue to use BlockManagerV1 and pass because this env gets set. I think we can remove it.